src/lib.rs

```diff
@@ -45,7 +46,7 @@ pub const DEFAULT_N_SHARDS: usize = 16;
 /// for lower contention when accessing concurrently.
 pub struct SymbolTable<const N: usize = DEFAULT_N_SHARDS, S = DeterministicHashBuilder> {
     build_hasher: S,
-    shards: [Mutex<Shard>; N],
+    shards: [CachePadded<Mutex<Shard>>; N],
```

Comment:
Can we use `repr` to avoid the dependency on crossbeam? https://github.com/crossbeam-rs/crossbeam-utils/issues/10

Reply:
Two reasons why I opted for CachedPadded. 

>  It is much simpler to just annotate your struct with #[repr(align(64))] than to deal with a wrapper type.

Annotating Shard wouldn't have the same behavior as forcing alignment for `Mutex<Shard>`. We need the new type pattern to get the right behavior. 

> dependency on crossbeam
It isn't as simple as just using `repr(align(64))` for all platforms. Fetch/MESI details vary between CPUs. This doesn't change very often, but I expect [crossbeam to be more likely to be kept up to date](https://docs.rs/crossbeam-utils/0.8.19/src/crossbeam_utils/cache_padded.rs.html#80) than if we copied its implementation. 

If you still would prefer not taking the dependency, I can opt for the somewhat conservative `repr(align(128))` on a newtype.

Reply:
Fair enough. Can you reduce the dependency to crossbeam_utils, the subcrate?

Reply:
Done